python/ngen_conf/src/ngen/config/hydrofabric.py

```diff
@@ -1,74 +1,35 @@
 
 from typing import List, Literal
 from pydantic import BaseModel, validator
-from geojson_pydantic import Feature, Point,  MultiPolygon
+from geojson_pydantic import FeatureCollection, Point,  MultiPolygon
+import re
 
-def check_in_accepted(val, accepted):
-    if val not in accepted:
-        raise ValueError(f'Not an acceptable option. Options are {accepted}')
+# match str that ends with a '-' followed by digits
+_TOID_REGEX_STR = ".\D+-\d+$"
```

Comment:
I think we want `"^\w+-\d+$"` here. `\D` captures any non-digit character, including `-`. `\w` will only capture word characters (including `_`).